pyrefly/lib/alt/class/class_field.rs

```diff
@@ -504,8 +554,12 @@ fn bind_instance_attribute(
             Some(make_bound_method(instance, attr).into_inner()),
             instance.class.dupe(),
         ),
-        attr if is_class_var || readonly => {
-            Attribute::read_only(make_bound_method(instance, attr).into_inner())
+        attr if is_class_var => Attribute::read_only(
+            make_bound_method(instance, attr).into_inner(),
+            ReadOnlyReason::Inherited,
```

Comment:
I think the guard is saying, this attribute is a ClassVar, so it can't be set from the instance

see https://typing.python.org/en/latest/spec/class-compat.html#classvar